packages/flutter_driver/lib/src/driver.dart

```diff
@@ -251,6 +266,14 @@ class FlutterDriver {
     return response['response'];
   }
 
+  void _logCommunication(String message)  {
+    if (_printCommunication)
+      _log.info(message);
+    f.File file = fs.file(p.join(testOutputsDirectory, 'flutter_driver_commands.log'));
```

Comment:
if you have multiple `flutter_driver` instances running on the same filesystem, this will result in corruption. Should we attempt to `flock()` the file while we're writing to it?